packages/react/test/element.spec.js

```diff
@@ -149,11 +149,17 @@ describe('element.applyProps', () =>
     test('AnimatedSprite.applyProps with images prop exists', () =>
     {
         const element = createElement(TYPES.AnimatedSprite, { images: ['./image.png'] });
-
         expect(element).toHaveProperty('applyProps');
         expect(spy).lastCalledWith('./image.png');
     });
 
+    test('AnimatedSprite.applyProps with updated image props', () =>
+    {
+        const element = createElement(TYPES.AnimatedSprite, { images: ['./image.png'] });
+        const changed = element.applyProps(element, { images: './image.png' }, { images: './new-image.png' });
```

Comment:
Correct me if I am wrong, but shouldn't this test be sending Arrays instead of Strings on line 159?

Reply:
Good catch - I've fixed this.  I'm not sure why I can't reproduce failures locally 🤔